src/emmy/abstract/function.cljc

```diff
@@ -157,12 +157,12 @@
   [f]
   (instance? Function f))
 
-(defn- name
-  "Returns the `-name` field of the supplied [[Function]] object. Errors if any
+(defn- f-name
```

Comment:
If we change the internal name can we keep this the same, or does that mess with what you're trying to do? I have a mild preference for taking advantage of namespaces, so users can write `function/name` vs `function/f-name`... but not if there is some clash with a JS thing here that we want to avoid.

Reply:
That's a great idea, thank you for pointing it out. The internal property name is an implementation detail, thank you for reminding me of that.